src/lib/chips/_chips-theme.scss

```diff
@@ -1,5 +1,11 @@
 @import '../core/theming/palette';
 @import '../core/theming/theming';
+@import '../core/typography/typography-utils';
+
+// TODO(crisbeto): these values don't correspond to any of the typography breakpoints.
```

Comment:
We might have to invent our own; the spec often paints broad strokes that don't capture the specific details for each component